app/routes/messages.rb

```diff
@@ -0,0 +1,24 @@
+Mumukit::Platform.map_organization_routes!(self) do
+  post '/courses/:course/messages' do
+    authorize! :teacher
+    message = json_body[:message]
+    message[:sender] = current_user.uid
+    assignment = Assignment.find_by!(with_organization_and_course 'exercise.eid': json_body[:exercise_id], 'student.uid': json_body[:uid])
+    assignment.add_message! message, json_body[:submission_id]
+    assignment.notify_message! message, json_body[:submission_id]
+    {status: :created, message: Message.new(message)}
+  end
+  get '/courses/:course/guides/:organization/:repository/:uid/:exercise_id/messages' do
```

Comment:
I use seccond aproach.

We should change labo to call this new route 
`<classroom-api>/api/guides/:organization/:repository/:exercise_id/student/:uid/messages`